src/react/deckgl.js

```diff
@@ -126,6 +128,18 @@ export default class DeckGL extends React.Component {
       );
     }
 
+    // TODO: add handlers on demand at runtime, not all at once on init
+    this.eventManager = createEventManager(canvas, {})
+    .on({
+      click: this._onClick,
```

Comment:
Per the conversation last week about supporting something like Mapbox's `queryRenderedFeatures`, I think we'll need to support both. E.g. an app dev might want an `onHover` callback that lets the app know when the hover target changes, but might also want to implement marquee selection manually with an `onMouseMove` handler that queries features within the marquee.